test/functorch/test_vmap.py

```diff
@@ -1057,6 +1057,20 @@ def f(x, y):
         expected = torch.mv(y, torch.ones(2)).view(3, 1, 1) + x
         self.assertEqual(out, expected)
 
+    def test_decomposition_under_python_dispatcher(self):
+        # This test will raise an error if the vmap fallback gets invoked.
+        # Here we test that decomps registered to FuncTorchBatchedDecomposition
+        # are respected by the Python Dispatcher.
+        t = torch.ones(3, 3) * 5
+        with warnings.catch_warnings(record=True) as wa:
+            warnings.simplefilter('always')
+            with EnableVmapFallbackWarnings():
```

Comment:
Wouldn't it be easier to just use `DisableVmapFallbackWarnings` from common_utils. The test would just fail automatically under that context without the fix.

Reply:
Thanks, that works. I forgot that that existed